tests/regionsearch/test_backend.py

```diff
@@ -0,0 +1,96 @@
+"""Tests for the Backend classes."""
+
+import numpy as np
+import pytest
+from astropy import units as u  # type: ignore
+from astropy.coordinates import (  # type: ignore
+    Angle,
+    Distance,
+    EarthLocation,
+    Latitude,
+    Longitude,
+)
+from astropy.time import Time  # type: ignore
+
+from kbmod.regionsearch import backend, indexers, utilities
+from kbmod.regionsearch.region_search import Filter
+
+
+def test_observationlist_init():
+    """Tests the ObservationList backend's constructor."""
+    data = utilities.RegionSearchClusterData(clustercnt=5, samplespercluster=10, removecache=True)
+
+    ra = data.observation_pointing.ra
+    dec = data.observation_pointing.dec
+    time = data.observation_time
+    location = data.observation_geolocation
+    fov = np.ones([data.rowcnt]) * Angle(1, "deg")
+    observation_identifier = data.cluster_id
+    b = backend.ObservationList(ra, dec, time, location, fov, observation_identifier)
+    assert all(b.observation_ra == ra)
+    assert all(b.observation_dec == dec)
+    assert all(b.observation_time == time)
+    assert all(b.observation_location == location)
+    assert all(b.observation_fov == fov)
+    assert all(b.observation_identifier == observation_identifier)
+
+
+def test_observationlist_consistency():
+    """Tests the ObservationList backend's constructor raise error if all elements are not equal in length."""
+    data = utilities.RegionSearchClusterData(clustercnt=5, samplespercluster=10, removecache=True)
+
+    ra = data.observation_pointing.ra
+    dec = data.observation_pointing.dec
+    time = data.observation_time
+    location = data.observation_geolocation
+    fov = Angle(1, "deg")
+    observation_identifier = data.cluster_id
+    with pytest.raises(ValueError):
+        backend.ObservationList(ra, dec, time, location, fov, observation_identifier)
+
+
+def test_observationlist_partition():
+    """Tests the ObservationList backend with partition indexer."""
+
+    # Compose a region search
+    class TestRegionSearchList(backend.ObservationList, indexers.PartitionIndexer):
+        """A test class for the ObservationList Backend with Partition indexer."""
+
+        def __init__(self, **kwargs) -> None:
+            super().__init__(**kwargs)
+
+    data = utilities.RegionSearchClusterData(clustercnt=5, samplespercluster=10, removecache=True)
+    ra = data.observation_pointing.ra
+    dec = data.observation_pointing.dec
+    time = data.observation_time
+    location = data.observation_geolocation
+    fov = np.ones([data.rowcnt]) * Angle(2, "deg")
+    observation_identifier = np.array([f"file:epyc/observations/{i:04d}" for i in range(data.rowcnt)])
```

Comment:
This isn't actually using data from epyc.  It is just using the name epyc in the observation_identifier. Is that correct?